code/modules/mob/dead/dead.dm

```diff
@@ -64,6 +64,7 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 	set category = "OOC"
 	set name = "Server Hop!"
 	set desc= "Jump to the other server"
+	set hidden = TRUE
```

Comment:
No this is not the way it works, remove the entry from the config file, dont hide the verb, that makes impossible to have a second server again unless the code is edited